src/App.tsx

```diff
@@ -8,37 +8,89 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { getUsers, getPosts } from './api';
+import { Post } from './types/Post';
+import { ErrorTypes } from './types/ErrorAction';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorType, setErrorType] = useState<ErrorTypes | null>(null);
+  const [isSelectedUser, setSelectedUser] = useState(false);
```

Comment:
```suggestion
  const [isSelectedUser, setIsSelectedUser] = useState(false);
```
setter name should be the same as the state variable name